portfolio/src/main/java/com/google/sps/servlets/NewServlet.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+
+
+@WebServlet("/welcome")
+public class NewServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+
+    ArrayList<String> quotes = new ArrayList<String>();
+    quotes.add("Do not give up without trying");
+    quotes.add(" I love gaming");
```

Comment:
I notice that all but the first have a leading space. Are the spaces for a reason? My best guess is that they have to do with producing/rendering the HTML. Generally, if the response from an API endpoint needs to be adapted for a specific use case, those adaptations would be made client-side (e.g. in JavaScript), rather than server-side where it would impact all consumers of the endpoint.